Dockerfile

```diff
@@ -22,7 +22,7 @@
 #
 
 # The Debian image that should be used as the basis for the guacd image
-ARG DEBIAN_BASE_IMAGE=buster-slim
+ARG DEBIAN_BASE_IMAGE=bullseye-slim
```

Comment:
Rather than referencing bullseye directly for `DEBIAN_BASE_IMAGE` and `DEBIAN_RELEASE`, would it be better to reference Debian stable? I'd think that would allow the nightly rebuilds of the image to pick up on distro updates even if there has not been a corresponding Guacamole update.

Reply:
Good idea, will do.